### PR TITLE
Filter sys schema on MySQL

### PR DESCRIPTION
**Pull request by lkm**
On MySQL 8 has a new sys schema that's used in conjunction with the performance schema, we should hide this as well. See this for details https://dev.mysql.com/doc/refman/8.0/en/sys-schema.html

### CODE REVIEW

**Comment by ebyhr** [comment]
@findepi Do you think we can just hide this schema? The `sys` schema has been bundled since version 5.7 (not 8) by default.

**Comment by findepi** [comment]
I think it's not unreasonable to hide it. I don't have a strong opinion though.

File: presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java [hunk 1]
```diff
@@ -185,6 +185,11 @@ protected boolean filterSchema(String schemaName)
         if (schemaName.equalsIgnoreCase("mysql")) {
             return false;
         }
+
+        if (schemaName.equalsIgnoreCase("sys")) {
+            return false;
+        }
+
```
Comment (ebyhr) [inline presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java:192]: Merge into the above if condition at L185.

**Review by ebyhr — APPROVED** [review]
Please remove ` 8` from commit title since it's not specific to the version.

**Comment by lkm** [comment]
Thanks @ebyhr, requested change made.

File: presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java [hunk 2]
```diff
@@ -182,9 +182,11 @@ public MySqlClient(BaseJdbcConfig config, ConnectionFactory connectionFactory, T
     @Override
     protected boolean filterSchema(String schemaName)
     {
-        if (schemaName.equalsIgnoreCase("mysql")) {
+        if (schemaName.equalsIgnoreCase("mysql")
+            || schemaName.equalsIgnoreCase("sys")) {
             return false;
         }
+
```
Comment (ebyhr) [inline presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java:189]: nit: Please revert unrelated change.

File: presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java [hunk 3]
```diff
@@ -182,7 +182,8 @@ public MySqlClient(BaseJdbcConfig config, ConnectionFactory connectionFactory, T
     @Override
     protected boolean filterSchema(String schemaName)
     {
-        if (schemaName.equalsIgnoreCase("mysql")) {
+        if (schemaName.equalsIgnoreCase("mysql")
+            || schemaName.equalsIgnoreCase("sys")) {
```
Comment (ebyhr) [inline presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java:186]: Please fix indentation as below. This is the CI failure cause. 
```suggestion
                || schemaName.equalsIgnoreCase("sys")) {
```

Code style exist here: https://github.com/prestosql/presto/blob/master/DEVELOPMENT.md#code-style

**Comment by lkm** [comment]
@ebyhr Pushed changes now. Apologies for forgetting this

**Comment by ebyhr** [comment]
Merged, thanks!
